ci-operator/config/opendatahub-io/openvino_model_server/opendatahub-io-openvino_model_server-new_release.yaml

```diff
@@ -53,11 +59,11 @@ resources:
       memory: 200Mi
   model-server-main-cuda-intel-gpu:
     limits:
-      cpu: "12"
+      cpu: "16"
```

Comment:
@heyselbi, @Xaenalt and @rpancham, I have bumped the resource usage and set limits equal to requests